edge/pkg/metamanager/process.go

```diff
@@ -253,11 +253,22 @@ func (m *metaManager) processUpdate(message model.Message) {
 }
 
 func (m *metaManager) processPatch(message model.Message) {
+	isSendToCloud := true
+	_, resType, _ := parseResource(&message)
+	if resType == model.ResourceTypePodPatchEdge {
+		isSendToCloud = false
+		resource := strings.Replace(message.GetResource(),
+			constants.ResourceSep+model.ResourceTypePodPatchEdge+constants.ResourceSep,
+			constants.ResourceSep+model.ResourceTypePodPatch+constants.ResourceSep, 1)
+		message.SetResourceOperation(resource, message.GetOperation())
+	}
 	if err := m.handleMessage(&message); err != nil {
 		feedbackError(err, message)
 		return
 	}
-	sendToCloud(&message)
```

Comment:
```
if !IsConnect {
   feedbackError(connect.ErrConnectionLost, message)
}
sendToCloud(&message)
```

Is it possible to use this to replace the modification in edge/pkg/metamanager/client/pod.go and here?